core/vm/errors.go

```diff
@@ -36,6 +36,7 @@ var (
 	ErrGasUintOverflow          = errors.New("gas uint64 overflow")
 	ErrInvalidCode              = errors.New("invalid code: must not begin with 0xef")
 	ErrNonceUintOverflow        = errors.New("nonce uint64 overflow")
+	errStopToken                = errors.New("stop token")
```

Comment:
Why is this lowercase?

Reply:
Lowercase names are not visible outside of a go package

Reply:
Is this name fine?

Reply:
I think it's ok. Perhaps it should be changed like this

```suggestion
	// errStopToken is an internal stop token which is never returned to outside callers.
	errStopToken                = errors.New("stop token")
```

Reply:
Added.